ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -257,7 +257,7 @@ def get_workflow(dict_configs, cdump='gdas'):
 
     tasks = []
 
-    if app in ['S2S', 'S2SW']:
+    if app[0:3] in ['S2S']:
```

Comment:
```suggestion
    if 'S2S' in app:
```
Any downside to that?

Reply:
I don't think so. Still not used to being able to do things like that with strings.

Reply:
Updated